.github/workflows/on-push-master.yml

```diff
@@ -19,3 +19,9 @@ jobs:
           if [ $(npm diff | wc -w) -gt 0 ]; then npm version patch && npm publish && echo "✨ The @polygon-nightfall/common-files package sucessfully published to npm"; else echo "✨ The @polygon-nightfall/common-files package is up to date"; fi
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - uses: stefanzweifel/git-auto-commit-action@v4
```

Comment:
If it OK for this step to happen even if the `npm diff` produces no changes? Won't it produce an empty commit? Did we check this?

Reply:
Yes, I'm sure because this step will run only if the `common-files/package.json` gets modified that will only happen if `npm version patch` runs that in which case there will be a diff.

Reply:
OK I see, nice !